ui_testing/pages/order_page.py

```diff
@@ -473,3 +513,11 @@ def get_order_id(self):
     def navigate_to_analysis_tab(self):
         self.base_selenium.click('order:analysis_tab')
         self.sleep_small()
+
+    def get_random_main_order_with_related_sub_orders_data(self):
```

Comment:
this function needs to be moved to orders_page, since it is related to orders table page not the form page